Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -303,6 +304,21 @@ def load_ipv8_overlays(self):
 
             self.ipv8.strategies.append((RandomWalk(self.market_community), 20))
 
+        # Popular Community
+        if self.session.config.get_popular_community_enabled() or True:
```

Comment:
Should this line always evaluate to True?

Reply:
Thanks! good catch